Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -271,13 +273,14 @@ namespace AZ
                                     // There are ways to write this without [[maybe_unused]], but they are brittle.
                                     // For example, using #else could cause a bug where the function's parameter
                                     // is changed in #ifdef but not in #else.
-                                    [[maybe_unused]] const uint32_t drawPacketCount=AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, *worklistData->m_view);
+                                    [[maybe_unused]]
+                                    const uint32_t drawPacketCount = AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, *worklistData->m_view);
```

Comment:
Why is the `[[maybe_unused]]` on the previous line?
It works, but it is unusual to put it on a line by itself.

Reply:
It was a long line and [[maybe_unused]] was just making it longer/harder to read. No real reason other than that